_posts/2010-06-01-odata-workshops-in-raleigh-charlotte-atlanta-chicago-and-nyc.html

```diff
@@ -12,7 +12,7 @@
 ---
 <div>
 <p>Chris (aka <a href="http://www.chriswoodruff.com/">Woody</a>) Woodruff has organized a series of <a href="http://www.chriswoodruff.com/index.php/2010/05/11/odata-blowout-in-june-8-odata-workshops-in-raleigh-charlotte-atlanta-chicago-and-nyc/"> OData workshops</a> to compliment the official <a href="http://odata.org/roadshow">OData Roadshow</a>.</p>
-<p>This is highly recommended. So if for whatever reason you can't make it to one of the OData Roadshow events, or you just can't get enough <a href="http://odata.org/">OData</a>, see if you can get along to one of Chris' workshops.</p>
+<p>This is highly recommended. So if for whatever reason you can't make it to one of the OData Roadshow events, or you just can't get enough <a href="/">OData</a>, see if you can get along to one of Chris' workshops.</p>
 <p>The workshops start in Raleigh tomorrow and finish in NYC on June 28th.</p>
 <p><a href="http://www.chriswoodruff.com/index.php/2010/05/11/odata-blowout-in-june-8-odata-workshops-in-raleigh-charlotte-atlanta-chicago-and-nyc/"> Learn more and register here</a>.</p>
```

Comment:
Broken link http://www.chriswoodruff.com/index.php/2010/05/11/odata-blowout-in-june-8-odata-workshops-in-raleigh-charlotte-atlanta-chicago-and-nyc

Reply:
@KenitoInc Fixing dead links that point to external resources was beyond the scope of this pull request. In any event, I didn't have a valid alternative I could replace them with.